src/sub-blocks/Content/__tests__/Content.test.tsx

```diff
@@ -0,0 +1,147 @@
+import {testCustomClassName} from '../../../../test-utils/shared/common';
+import {
+    testContentByDefault,
+    testContentWithAdditionalInfo,
+    testContentWithButtons,
+    testContentWithCentered,
+    testContentWithColSize,
+    testContentWithLinks,
+    testContentWithList,
+    testContentWithSize,
+    testContentWithText,
+    testContentWithTheme,
+    testContentWithTitle,
+} from '../../../../test-utils/shared/content';
+import {GridColumnSizesType} from '../../../grid/types';
+import {ContentSize, ContentTheme} from '../../../models/constructor-items/common';
+import {getCommonQa} from '../../../utils/blocks';
+import Content, {ContentProps} from '../Content';
+
+const contentData: ContentProps = {
+    title: 'title',
+    text: 'text',
+    additionalInfo: 'additional info',
+    links: [{url: 'https://example.com', theme: 'normal'}],
+    buttons: [{url: 'https://example.com', text: 'button'}],
+    centered: true,
+    list: [
+        {
+            icon: 'https://storage.yandexcloud.net/cloud-www-assets/constructor/storybook/images/img-gray.png',
+            title: 'list title',
+            text: 'list text',
+        },
+    ],
+    qa: 'content',
+};
+
+const colSizesArray: GridColumnSizesType[] = [
+    {all: 9, lg: 8, md: 7, sm: 6},
+    {all: 5, lg: 4, md: 3, sm: 2},
+];
+
+const qas = getCommonQa(contentData.qa, ['link', 'list']);
+
+describe('Content', () => {
+    test('Render by default', async () => {
+        testContentByDefault({
+            component: Content,
```

Comment:
why do we need to pass `Content` component to every test instead of import it in `test-utils/shared/content`?

Reply:
good point